src/factory/queuing-factory.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.const redlock = require('redlock');
+
+const config = require('painless-config');
```

Comment:
By convention we have been alphabetizing the requires.  Typically based on the const name (as opposed to the module name)